build/_data/candidates/oakland/2016-11-08/jody-london.json

```diff
@@ -25,10 +25,10 @@
     "total_contributions": 22344.0,
     "total_expenditures": 23941.0,
     "total_loans_received": 0.0,
-    "total_supporting_independent": 2690.73,
+    "total_supporting_independent": 5780.41,
     "support_list": [
       {
-        "Total": 2690.73,
+        "Total": 5780.41,
```

Comment:
Some of these numbers are significantly different! Just checking that you have considered that.

Reply:
I looked at some of them.  Our latest_only algorithm really does not work well given the variety of ways people seem to file.